### PR TITLE
fix: CLIN-2325 - download TSV

### PR DESCRIPTION
**Pull request by creativeyann17**
# FIX

## Description

[[JIRA LINK]](https://ferlab-crsj.atlassian.net/browse/CLIN-2325)

Acceptance Criterias

## Validation

- [ ] Code Approved
- [ ] QA Done
- [ ] Design/UI Approved from design

## QA

Steps to validate
Url (storybook, ...)
...

### CODE REVIEW

File: src/views/Prescriptions/Entity/GenericCoverage/index.tsx [hunk 1]
```diff
@@ -227,7 +226,7 @@ const Index = ({ downloadFile }: any) => {
               )}
               initialColumnState={initialColumnState}
               scroll={dimension}
-              dataSource={data}
+              dataSource={data.map((i) => ({ ...i, key: `${i[VARIANT_KEY]}` }))}
```
Comment (creativeyann17) [inline src/views/Prescriptions/Entity/GenericCoverage/index.tsx:229]: https://ferlab-crsj.atlassian.net/browse/CLIN-2330

File: src/components/Download.tsx [hunk 2]
```diff
@@ -46,60 +44,74 @@ const DownloadTSVWrapper = ({
...
+    );
+
+    return sendRequestWithRpt<{ data: any }>({
+      method: 'POST',
+      url: ARRANGER_API_PROJECT_URL,
+      data: {
+        query: query.loc ? query.loc!.source.body : query,
+        variables: {
+          ...queryVariables,
+          searchAfter: undefined,
```
Comment (creativeyann17) [inline src/components/Download.tsx:79]: https://ferlab-crsj.atlassian.net/browse/CLIN-2331

File: src/views/Prescriptions/Search/index.tsx [hunk 3]
```diff
@@ -131,12 +131,14 @@ const PrescriptionSearch = (): React.ReactElement => {
   // query is always done, unfortunately but response size is limited if nothing to download
   const prescriptionsToDownload = usePrescription({
     ...prescriptionsQueryVariables,
+    searchAfter: undefined,
```
Comment (creativeyann17) [inline src/views/Prescriptions/Search/index.tsx:134]: https://ferlab-crsj.atlassian.net/browse/CLIN-2331

File: src/views/Prescriptions/Search/index.tsx [hunk 4]
```diff
@@ -131,12 +131,14 @@ const PrescriptionSearch = (): React.ReactElement => {
...
     first: downloadPrescriptionKeys.length > 0 ? prescriptions.total : 0,
   });
 
   // query is always done, unfortunately but response size is limited if nothing to download
   const sequencingsToDownload = useSequencingRequests({
     ...sequencingsQueryVariables,
+    searchAfter: undefined,
```
Comment (creativeyann17) [inline src/views/Prescriptions/Search/index.tsx:141]: https://ferlab-crsj.atlassian.net/browse/CLIN-2331

File: src/views/Prescriptions/utils/export.ts [hunk 5]
```diff
@@ -60,72 +59,6 @@ function getLeafNodes(obj: any): string {
   return String(Array.from(new Set(traverse([], obj))).join(JOIN_SEP));
 }
 
-export const buildVariantsDownloadCount = (
```
Comment (creativeyann17) [inline src/views/Prescriptions/utils/export.ts:63]: Not used anymore

**Review by ethienneroy — APPROVED** [review]

**Review by kstonge — APPROVED** [review]

**Pull request merged**